src/main/java/com/github/jreddit/action/FlairActions.java

```diff
@@ -0,0 +1,674 @@
+package com.github.jreddit.action;
+
+import com.github.jreddit.entity.User;
+import com.github.jreddit.retrieval.ActorDriven;
+import com.github.jreddit.utils.ApiEndpointUtils;
+import com.github.jreddit.utils.restclient.Response;
+import com.github.jreddit.utils.restclient.RestClient;
+
+/**
+ * Flair actions for posts and users
+ *
+ * @author Ryan Delaney (Vitineth)
+ * @since 03/01/2015
+ */
+public class FlairActions implements ActorDriven {
+
+    private final RestClient restClient;
+    private User user;
+
+    /**
+     * Constructor. Global default user (null) is used.
+     *
+     * @param restClient REST Client instance
+     */
+    public FlairActions(RestClient restClient) {
+        this.restClient = restClient;
+        this.user = null;
```

Comment:
if you don't specify the __this.user = null__, it will be null when the class is instantianciated, so this line is redundant